src/bit-systems/mixer-animation.ts

```diff
@@ -0,0 +1,42 @@
+import { defineQuery, exitQuery } from "bitecs";
+import { AnimationClip, AnimationMixer } from "three";
+import {
+  MixerAnimatable,
+  MixerAnimatableData,
+  LoopAnimation
+} from "../bit-components";
+import { cleanupMixerAnimatable } from "../inflators/mixer-animatable";
+import { cleanupLoopAnimation } from "../inflators/loop-animation";
+import { HubsWorld } from "../app";
+
+// TODO: This type declaration should be in bit-components
+type MixerAnimatableData = Map<number, {
+  mixer: AnimationMixer,
+  animations: Array<AnimationClip>
+}>;
+
+const mixerAnimatableQuery = defineQuery([MixerAnimatable]);
+const exitedMixerAnimatableQuery = exitQuery(mixerAnimatableQuery);
+
+const loopAnimationQuery = defineQuery([LoopAnimation]);
+const exitedLoopAnimationQuery = exitQuery(loopAnimationQuery);
+
+// Question: Who should have AnimationMixer?
+//           MixerAnimatable component or Scene/App?
+
+export function mixerAnimationSystem(world: HubsWorld, delta: number) : void {
+  mixerAnimatableQuery(world).forEach(eid => {
```

Comment:
Before this loop I think we can add an enteryQuery here for `MixerAnimatable` where we will actually set up the mixer. Then we can have an enter query for `[MixerAnimatable,LoopAnimation]` to set up the clip for the looping animation and add it to the mixer.
Then by the time we hit this loop everything is guaranteed to be set up.